sqlglot/generator.py

```diff
@@ -46,6 +46,8 @@ class Generator:
             The default is on the smaller end because the length only represents a segment and not the true
             line length.
             Default: 80
+        default_order: The default order to use in the ORDER BY clause if ASC / DESC are omitted.
```

Comment:
i don't think this should be passed in a string, just a boolean

Reply:
sounds good with me, will update it so it expects an optional boolean and `None` will stand for `""`.

Reply:
yea I was thinking something like `sort_asc` or something

Reply:
Actually, I don't think I under what `True` or `False` would mean here.

Does `True` mean "always render the order" and `False` mean "render only if `DESC`?

Reply:
If the arg was named `default_order_is_asc`, then `True` means `ASC` is printed in the "default" `ORDER BY`, `False` means `DESC` is printed and `None` means it's left as is, i.e `ORDER BY a`.

I'm actually taking a closer look again to see if I can just preserve `ASC` / `DESC` as you suggested in the issue. I tried to do this initially but we have a lot of test failures and I need to verify it's a safe change to make.

Reply:
https://github.com/tobymao/sqlglot/pull/2256/commits/198099d80a671684680bbd89039a0c5bfefe998b